src/MOI_wrapper.jl

```diff
@@ -418,3 +443,63 @@ end
 function MOI.modify(model::Optimizer, ci::CI{F, S}, chg::MOI.AbstractFunctionModification) where {F<:MOI.VectorAffineFunction{Float64}, S <: SUPPORTED_VECTOR_SETS}
     MOI.modify(model.optimizer, ci, chg)
 end
+
+"""
+    Method to differentiate optimal solution `x`, `y`, `s`
+"""
+function backward_conic!(model::Optimizer)
+    if MOI.get(model, MOI.TerminationStatus()) in [MOI.LOCALLY_SOLVED, MOI.OPTIMAL]
+        @assert MOI.get(model.optimizer, MOI.SolverName()) == "SCS"
```

Comment:
is this verification necessary?

Reply:
actually not :sweat_smile: i didn't knew if any other solver also expresses the matrices A,b,c as SCS.jl does